spot_ros2_control/examples/src/joint_command_passthrough.cpp

```diff
@@ -18,18 +18,21 @@
 class JointCommandPassthrough : public rclcpp::Node {
  public:
   JointCommandPassthrough() : Node("joint_passthrough") {
+    std::string robot_namespace = declare_parameter("robot_namespace", "Spot");
+    std::string joint_state_topic = robot_namespace + "/" + declare_parameter("joint_state_topic", "joint_states");
```

Comment:
Can you default this to empty and only have the "/" if the robot_namespace is not empty? That is the default for users who don't have multiple spots.